integration/solana/account_data.spec.ts

```diff
@@ -0,0 +1,261 @@
+// SPDX-License-Identifier: Apache-2.0
+
+
+import {loadContractAndCallConstructor, newConnectionAndPayer} from "./setup";
+import {Connection, Keypair, PublicKey} from "@solana/web3.js";
+import {
+    approveChecked,
+    AuthorityType,
+    createMint,
+    getAccount,
+    getOrCreateAssociatedTokenAccount, mintTo,
+    setAuthority
+} from "@solana/spl-token";
+import { publicKey, u64, bool } from '@solana/buffer-layout-utils';
+import { u32, u8, struct } from '@solana/buffer-layout';
+import expect from "expect";
+import {Program} from "@coral-xyz/anchor";
+import assert from "assert";
+import exp from "constants";
+
+describe('Deserialize account data', function () {
+    this.timeout(500000);
+
+    interface RawMint {
+        mintAuthorityOption: 1 | 0;
+        mintAuthority: PublicKey;
+        supply: bigint;
+        decimals: number;
+        isInitialized: boolean;
+        freezeAuthorityOption: 1 | 0;
+        freezeAuthority: PublicKey;
+    }
+
+    const MintLayout = struct<RawMint>([
+        u32('mintAuthorityOption'),
+        publicKey('mintAuthority'),
+        u64('supply'),
+        u8('decimals'),
+        bool('isInitialized'),
+        u32('freezeAuthorityOption'),
+        publicKey('freezeAuthority'),
+    ]);
+
```

Comment:
Why are these definitions here? They're from `@solana/spl-token` (where they belong). Can't you use `getMint()` so we don't have to copy and paste this?

Reply:
I found a guide on the internet about the Solana library that misled me and made have these structures. I should have checked deeper the library before following the guide.

Reply:
Fixed now.